tripBackend/src/modules/productSchedule/commands/handlers/deleteProductScheduleByProductScheduleId.handler.ts

```
@@ -0,0 +1,71 @@
+import { DeleteProductScheduleByProductScheduleIdCommand } from '../implements';
+
+@CommandHandler(DeleteProductScheduleByProductScheduleIdCommand)
+export class DeleteProductScheduleByProductScheduleIdHandler
+	implements ICommandHandler<DeleteProductScheduleByProductScheduleIdCommand>
+{
+	constructor(private readonly productScheduleRepository: ProductScheduleRepository) {}
+
+	private checkSupplierPermissions(
+		supplier: SupplierInformationDto,
+		productSchedule: ProductScheduleEntity,
+	): void | HttpException {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)